src/main/java/com/uber/jenkins/phabricator/conduit/DifferentialClient.java

```diff
@@ -121,4 +142,8 @@ public JSONObject postComment(String revisionID, String message) throws ConduitA
     protected JSONObject callConduit(String methodName, Map<String, String> params) throws ConduitAPIException, IOException {
         return conduit.perform(methodName, params);
     }
+
+    protected JSONObject callConduit(String methodName, JSONObject params) throws ConduitAPIException, IOException {
```

Comment:
I assumed the `Map<String,String>` methods were written because they provided some sort of convenience.  Are you saying you would rather get rid of them and use `JSONObject` directly at all of the call sites?